bin/rr-worker

```diff
@@ -30,7 +30,13 @@ if ($loaded !== true) {
     );
 }
 
-global $argv;
+global $argv, $_ENV;
+
+$worker_class = isset($_ENV['RR_WORKER_CLASS']) && \is_string($_ENV['RR_WORKER_CLASS'])
```

Comment:
using global parameters is a bad idea, maybe getenv / filter_input

Reply:
CLI context =)

Reply:
_updated_